recipes/yarn.rb

```diff
@@ -198,15 +215,15 @@
     paths [
             "#{Chef::Config['file_cache_path']}/#{hopsExamplesSpark}", 
             "#{Chef::Config['file_cache_path']}/#{hopsExamplesFeaturestoreTour}", 
-            "#{Chef::Config['file_cache_path']}/#{hsfs_utils}",
+            "#{Chef::Config['file_cache_path']}/#{hsfs_utils_py}",
```

Comment:
Here we need to do the same also for the `hsfs_util_java` otherwise it won't work on hopsworks.ai